packages/datadog_flutter_plugin/README.md

```diff
@@ -1,12 +1,16 @@
 <p align="center">
-  <img src="dd_logo.png" width="200">
+  {{< img src="dd-logo.png" alt="Datadog logo" style="width:30%;">}}
```

Comment:
So... it looks like we might not be able to have this in for the README and out on the documentation site. By using this syntax (I'm guessing so the Docs site works), it breaks the README (and the result on pub.dev) so probably best to just remove it.

Reply:
Ok, I'm sorry that we can't replicate the image when we single source it into the docs 😫

Reply:
We can still keep the image in the main repo README (https://github.com/DataDog/dd-sdk-flutter#readme)!